### PR TITLE
Massive fact additions

### PR DESCRIPTION
**Pull request by Hazematman**
Slowly getting us closer to a new fact every day of the year

### CODE REVIEW

File: assets/lucashifacts [hunk 1]
```diff
@@ -100,3 +100,11 @@ lucashi walked into the fridge door and spilled vegetables all over himself
 lucashi once dated an ex-convict
 lucashi is a fully licensed motorcycle rider in Ontario
 lucashi has climbed to the peak of mt. Elbert the tallest moutain in the rocky mountains
+the only bone lucashi has ever broken is his wrist bone
+lucashi has trained new employees at CoreAVI
+lucashi has a tendency to have drinks poured all over his pant
```
Comment (karlding) [inline assets/lucashifacts]: `"pant" => "pants"`

File: assets/lucashifacts [hunk 2]
```diff
@@ -100,3 +100,11 @@ lucashi walked into the fridge door and spilled vegetables all over himself
...
+lucashi got so drunk at a work party once, he forgot his underwear in his coworkers car
```
Comment (karlding) [inline assets/lucashifacts]: Shouldn't this be `coworker's`?

File: assets/lucashifacts [hunk 3]
```diff
@@ -100,3 +100,11 @@ lucashi walked into the fridge door and spilled vegetables all over himself
...
+lucashi survived the tragic 5.0 magnitude earthquake that struck Ottawa in the summer of 2010
+lucashi at the request of his CEO, took shots with many corporate executives
```
Comment (karlding) [inline assets/lucashifacts]: comma after lucashi? Or mdashes

File: assets/lucashifacts [hunk 4]
```diff
@@ -100,3 +100,11 @@ lucashi walked into the fridge door and spilled vegetables all over himself
 lucashi once dated an ex-convict
 lucashi is a fully licensed motorcycle rider in Ontario
 lucashi has climbed to the peak of mt. Elbert the tallest moutain in the rocky mountains
+the only bone lucashi has ever broken is his wrist bone
+lucashi has trained new employees at CoreAVI
+lucashi has a tendency to have drinks poured all over his pants
+lucashi got so drunk at a work party once, he forgot his underwear in his coworker's car
```
Comment (karlding) [inline assets/lucashifacts]: I think it reads a bit nicer if it's like: lucashi once got so drunk at a work party that he forgot..

File: assets/lucashifacts (hunk 3 above)
Comment (Hazematman) [reply assets/lucashifacts]: So like "lucashi, at the request of his CEO, took ..." ?

**Review by karlding — APPROVED** [review]
LGTM

**Pull request merged**